docs/developers/func-file.md

```diff
@@ -72,15 +66,4 @@ if found in your local environment. If these vars are not found, local testing w
 `build` (optional) is an array of local shell calls which are used to help
 building the function. TODO: Deprecate this?
```

Comment:
Kind of a cool feature but I've never used it and have no idea what it can do or how it interacts with built in build behavior.  Are there any docs on this?